packages/@aws-cdk/aws-apigatewayv2/lib/common/stage.ts

```diff
@@ -66,6 +95,52 @@ export interface StageOptions {
    * @default - no throttling configuration
    */
   readonly throttle?: ThrottleSettings;
+
+  /**
+   * Whether or not to enable access logging.
+   *
+   * @default - false
+   */
+  readonly accessLogEnabled?: boolean;
+
+  /**
+   * Optionally, specify a log group that access log entries will be written to.
+   * @default - If not specified a log group will be created if access logs are enabled.
+   */
+  readonly accessLogGroupArn?: string;
+
+  /**
+   * The format and contents of an access log entry.
+   *
+   * @default - If not specified a the default fields logged will be set to
```

Comment:
Where did this default come from? Is there some documentation recommending this as the default?

Reply:
No, there's no recommendation for a default.  The actual default is to log nothing, which is an option that no is going to want.  The attributes I chose are most of the attributes that apply to both web socket and http apis.

Reply:
Are you OK with leaving the default in?

Reply:
Can we copy what is done in `apigateway`?

https://github.com/aws/aws-cdk/blob/b8c2abf06ba14f9c356864aba91180a55cbd9c3b/packages/@aws-cdk/aws-apigateway/lib/stage.ts#L43-L50

Reply:
I could, and I will if that's what you really want to do.  The thing that's done in the apigateway code is to not have a default and throw exceptions if/when the "must include contextRequestId" or "must include a destination" rules are violated.  The rational behind including a reasonable default is that users will at least be able to see basic access activity and error messages when they enable access logging without having to navigate that gauntlet, which to me seems like a much more user friendly experience.  I don't think that imitating the apigateway is the best way to go here, but you have the final say.  Do you want me to imitate the apigateway behavior for this or leave in the user friendly default?